scripts/initialize-celo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -euo pipefail
+
+KEEP_CORE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../../keep-core)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
```

Comment:
Same question as for `initialize.sh` - shouldn't we place it in `# Defaults, can be overwritten...` section?

Reply:
045db58